src/ainft.ts

```diff
@@ -141,12 +150,25 @@ export default class AinftJs {
     this.setSigner(signer);
   }
 
+  async open() {
+    const privateKey = this.ain.wallet.defaultAccount?.private_key;
+    if (privateKey) {
+      await this.ainize.login(privateKey);
+    } else {
+      await this.ainize.loginWithSigner();
+    }
+  }
+
+  async close() {
```

Comment:
could you add some tsdoc? e.g. 'Closes the session with the blockchain endpoint'

Reply:
done!